### PR TITLE
Review 0.0.3. - 3

### PR DESCRIPTION
**Pull request by compriab**
Changes:
- Food Names.
- Build Itens
- Skill Electricity

### CODE REVIEW

File: PT-BR/Game.po [hunk 1]
```diff
@@ -6934,21 +6934,21 @@ msgstr "Milho"
...
-msgstr "Ensopado de Mel"
+msgstr "Lombo na Mostarda e Mel"
 
 #. Key:	F32
 #. SourceLocation:	/Game/LocalizationText/ItemInfo.ItemInfo
 #: /Game/LocalizationText/ItemInfo.ItemInfo
 msgctxt "ItemInfo,F32"
 msgid "菜汤"
-msgstr "Sopa de Legumes"
+msgstr "Sopa de Verde"
```
Comment (Welenkens) [inline PT-BR/Game.po:6951]: Sopa Verde

File: PT-BR/Game.po [hunk 2]
```diff
@@ -6976,28 +6976,28 @@ msgstr "Salada de Frutas"
...
-msgstr "Ensopado de Mel"
+msgstr "Lombo ao Molho de Mostarda e Mel"
 
 #. Key:	F39
 #. SourceLocation:	/Game/LocalizationText/ItemInfo.ItemInfo
 #: /Game/LocalizationText/ItemInfo.ItemInfo
 msgctxt "ItemInfo,F39"
 msgid "浓郁炖肉"
-msgstr "Ensopado de Carne"
+msgstr "Feijoada Mineira"
```
Comment (Welenkens) [inline PT-BR/Game.po:7000]: Feijoada

File: PT-BR/Game.po [hunk 3]
```diff
@@ -7011,7 +7011,7 @@ msgstr "Batata"
 #: /Game/LocalizationText/ItemInfo.ItemInfo
 msgctxt "ItemInfo,F40"
 msgid "肉汤"
-msgstr "Caldo de Carne"
+msgstr "Caldo de Iscas de Carne"
```
Comment (Welenkens) [inline PT-BR/Game.po:7014]: Caldo de Carne

File: PT-BR/Game.po [hunk 4]
```diff
@@ -7088,14 +7088,14 @@ msgstr "Tomate"
...
-msgstr "Ensopado de Mel"
+msgstr "Caldo de Mel Gourmet"
 
 #. Key:	F51
 #. SourceLocation:	/Game/LocalizationText/ItemInfo.ItemInfo
 #: /Game/LocalizationText/ItemInfo.ItemInfo
 msgctxt "ItemInfo,F51"
 msgid "美味浓郁炖肉"
-msgstr "Ensopado Tradicional"
+msgstr "Feijoada Mineira Gourmet"
```
Comment (Welenkens) [inline PT-BR/Game.po:7098]: Feijoada Gourmet

File: PT-BR/Game.po [hunk 5]
```diff
@@ -7242,28 +7242,28 @@ msgstr "Morango"
 #: /Game/LocalizationText/ItemInfo.ItemInfo
 msgctxt "ItemInfo,F70"
 msgid "美味蔬菜肉罐头"
-msgstr "Lata de Carne Vegetal"
+msgstr "Lata de Picadinho com Legumes Gourmet"
```
Comment (Welenkens) [inline PT-BR/Game.po:7245]: Lata de **Carne** com Legumes Gourmet

File: PT-BR/Game.po [hunk 6]
```diff
@@ -7200,35 +7200,35 @@ msgstr "Garrafa de Plástico Grande"
...
+msgstr "Cantil"
 
 #. Key:	F66
 #. SourceLocation:	/Game/LocalizationText/ItemInfo.ItemInfo
-#: /Game/LocalizationText/ItemInfo.ItemInfo
+#: /Game/LocalizationText/ItemInfo.ItemInfo (Lata Verde)
 msgctxt "ItemInfo,F66"
 msgid "蔬菜肉罐头"
-msgstr "Lata de Carne Vegetal"
+msgstr "Lata de Picadinho (Chili Simples)"
```
Comment (Welenkens) [inline PT-BR/Game.po:7210]: Lata de Carne com Chili

File: PT-BR/Game.po [hunk 7]
```diff
@@ -6689,7 +6689,7 @@ msgstr "Laranja"
 #: /Game/LocalizationText/ItemInfo.ItemInfo
 msgctxt "ItemInfo,F120"
 msgid "肉罐头"
-msgstr "Lata de Carne"
+msgstr "Lata de Iscas de Carne"
```
Comment (Welenkens) [inline PT-BR/Game.po:6692]: Lata de Carne

File: PT-BR/Game.po [hunk 8]
```diff
@@ -7242,28 +7242,28 @@ msgstr "Morango"
...
-msgstr "Lata de Carne ao Molho de Mel"
+msgstr "Lombo na Mostarda e Mel Gourmet"
 
 #. Key:	F73
 #. SourceLocation:	/Game/LocalizationText/ItemInfo.ItemInfo
 #: /Game/LocalizationText/ItemInfo.ItemInfo
 msgctxt "ItemInfo,F73"
 msgid "美味浓郁肉罐头"
-msgstr "Lata de Carne"
+msgstr "Lata de Iscas de Carne Gourmet"
```
Comment (Welenkens) [inline PT-BR/Game.po:7266]: Lata de Carne Gourmet

File: PT-BR/Game.po [hunk 9]
```diff
@@ -12380,7 +12380,7 @@ msgstr "Metalurgia"
 #: /Game/LocalizationText/Screen.Screen
 msgctxt "UI,YJSW"
 msgid "永久死亡"
-msgstr "Permadeath"
+msgstr "Morte Permanente"
```
Comment (Welenkens) [inline PT-BR/Game.po:12383]: Morte Permanente **(Permadeath)**

**Review by Welenkens — CHANGES_REQUESTED** [review]

File: PT-BR/Game.po (hunk 1 above)
Comment (compriab) [reply PT-BR/Game.po:6951]: OK

File: PT-BR/Game.po (hunk 2 above)
Comment (compriab) [reply PT-BR/Game.po:7000]: Ok

File: PT-BR/Game.po (hunk 3 above)
Comment (compriab) [reply PT-BR/Game.po:7014]: OK

File: PT-BR/Game.po (hunk 6 above)
Comment (compriab) [reply PT-BR/Game.po:7210]: OK

File: PT-BR/Game.po (hunk 5 above)
Comment (compriab) [reply PT-BR/Game.po:7245]: OK

File: PT-BR/Game.po (hunk 7 above)
Comment (compriab) [reply PT-BR/Game.po:6692]: OK

File: PT-BR/Game.po (hunk 8 above)
Comment (compriab) [reply PT-BR/Game.po:7266]: OK

File: PT-BR/Game.po (hunk 9 above)
Comment (compriab) [reply PT-BR/Game.po:12383]: OK

**Pull request merged**